src/writing-policies/rego/gatekeeper/02-create-policy.md

```diff
@@ -0,0 +1,101 @@
+# Create a new policy
+
+Let's implement the same policy that [we wrote with Open Policy
+Agent](../open-policy-agent/02-create-policy.md): a policy that
+rejects a resource if it's targeting the `default` namespace.
+
+## Requirements
+
+As in the previous section, we will require the following tools:
+
+- `opa`
+- `kwctl`
+
+## The policy
+
+Since Gatekeeper is targeting Kubernetes, it has the freedom to be
+more handy in what the policy has to return.
+
+With Open Policy Agent we had to construct a whole `AdmissionReview`
+object as the response of our policy. With Gatekeeper, we only have to
+return none or more violations from our policy entrypoint.
+
+If no violations were reported, the request will be accepted. If one,
+or more violations were reported, the request will be rejected.
+
+We create a new folder, named `rego-policy`. Inside of it, we create a
+`policy.rego` file with contents:
+
+```rego
+package policy
+
+violation[{"msg": msg}] {
+        input.review.object.metadata.namespace == "default"
+        msg := "it is forbidden to use the default namespace"
+}
+```
+
+In this case, our entrypoint is `policy/violation`, and because of how
+Rego works, it can either have 1 violation: if the object to be
+reviewed is targeting the `default` namespace, or 0 violations
+otherwise.
```

Comment:
It doesn't feel natural to me to use `:` like that in the sentence.

I would replace everything with:

> In this case, our entrypoint is `policy/violation`, and because of how
Rego works, the policy can have the following outcomes:
> * return 1 `violation`: the object being reviewed is targeting the `default` namespace
> * return 0 violations: the object being reviewed is complaint with the policy